packages/flame/lib/src/extensions/aabb.dart

```diff
@@ -0,0 +1,6 @@
+import 'package:flame/extensions.dart';
+
+extension Aabb2Extension on Aabb2 {
+  /// Creates a [Rect] starting in [min] and going the [max]
+  Rect toRect() => Rect.fromLTRB(min.x, min.y, max.x, max.y);
```

Comment:
This doesn't seem to be used anywhere?

Reply:
I thought, here  https://github.com/flame-engine/flame/pull/1894#discussion_r967194766  you asked me to add this functionality, isn't it?..

Reply:
Ah yes, you are totally right!